pkg/util/export/etl/sqlWriter/sqlWriter.go

```diff
@@ -0,0 +1,268 @@
+// Copyright 2022 Matrix Origin
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sqlWriter
+
+import (
+	"context"
+	"database/sql"
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"sync"
+
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/matrixorigin/matrixone/pkg/logutil"
+	"github.com/matrixorigin/matrixone/pkg/util/export/table"
+	"go.uber.org/zap"
+)
+
+const PACKET_LARGE_ERROR = "packet for query is too large"
+
+var _ SqlWriter = (*BaseSqlWriter)(nil)
+
+// BaseSqlWriter SqlWriter is a writer that writes data to a SQL database.
+type BaseSqlWriter struct {
+	db      *sql.DB
+	address string
+	ctx     context.Context
+	dbMux   sync.Mutex
+}
+
+type SqlWriter interface {
+	table.RowWriter
+	WriteRows(rows string, tbl *table.Table) (int, error)
+}
+
+func (sw *BaseSqlWriter) GetContent() string {
+	return ""
+}
+func (sw *BaseSqlWriter) WriteRow(row *table.Row) error {
+	return nil
+}
+
+func generateInsertStatement(records [][]string, tbl *table.Table) (string, int, error) {
```

Comment:
I am not sure this is the right way to do the insert.    I would rather do a prepared statement, then call insert as stmt.Execute(args...)